### ISSUE TITLE
unresponsive

### ISSUE THREAD

**Issue by chapus** [microsoft/vscode#184780]
Type: <b>Bug</b>

vs code is slow, when i go to running extensions this extensions appears with a message "unresponsive" Performance Issue, but no more details.

VS Code version: Code 1.79.0 (b380da4ef1ee00e224a15c1d4d9793e27c2b6302, 2023-06-07T14:26:35.552Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-10900KF CPU @ 3.70GHz (20 x 3696)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.92GB (27.27GB free)|
|Process Argv|. --crash-reporter-id 7445e071-e501-4b52-95eb-3191424218fc|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (97)</summary>

Extension|Author (truncated)|Version
---|---|---
laravel-extra-intellisense|ami|0.6.3
ng-template|Ang|16.0.0
vscode-django|bat|1.10.0
emojisense|bie|0.10.0
markdown-checkbox|bie|0.4.0
markdown-emoji|bie|0.3.0
markdown-footnotes|bie|0.1.1
markdown-image-size|bie|0.0.4
markdown-preview-github-styles|bie|2.0.2
vscode-intelephense-client|bme|1.9.5
vscode-tailwindcss|bra|0.9.11
dscodegpt|Dan|2.1.10
dart-code|Dar|3.66.0
flutter|Dar|3.66.0
vscode-eslint|dba|2.4.0
xslt-xpath|del|1.5.12
githistory|don|0.6.20
python-environment-manager|don|1.0.4
python-extension-pack|don|1.7.0
xml|Dot|2.5.1
prettier-vscode|esb|9.13.0
code-runner|for|0.12.0
copilot|Git|1.88.132
copilot-labs|Git|0.14.884
heygithub|Git|0.21.1441
vscode-pull-request-github|Git|0.66.0
go|gol|0.38.0
vscode-drawio|hed|1.6.6
beautify|Hoo|1.5.0
rest-client|hum|0.25.1
vscode-csharp-snippets|jor|1.1.0
vsc-python-indent|Kev|1.18.0
vscode-commitizen|Kni|1.1.0
mp-rest-client-generator-vscode-ext|Mic|0.1.2
mp-starter-vscode-ext|Mic|0.2.9
vscode-microprofile-pack|Mic|0.1.3
vscode-docker|ms-|1.25.1
vscode-language-pack-es|MS-|1.79.2023060709
csharp|ms-|1.25.9
vscode-kubernetes-tools|ms-|1.3.13
data-workspace-vscode|ms-|0.5.0
mssql|ms-|1.19.1
sql-bindings-vscode|ms-|0.4.0
sql-database-projects-vscode|ms-|1.1.1
python|ms-|2023.10.0
jupyter|ms-|2023.5.1001582324
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.15
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.295.0
remote-wsl|ms-|0.79.4
cpptools|ms-|1.15.4
hexeditor|ms-|1.9.11
vsliveshare|ms-|1.0.5873
composer-Intelephense|mua|1.0.1
vetur|oct|0.37.3
es6-css-minify|olb|3.3.3
vscode-versionlens|pfl|1.5.0
vscode-css-peek|pra|4.2.0
prisma|Pri|4.15.0
fabric8-analytics|red|0.3.10
java|red|1.19.0
vscode-commons|red|0.0.6
vscode-microprofile|red|0.7.1
vscode-quarkus|red|1.13.0
vscode-rsp-ui|red|0.24.7
vscode-server-connector|red|0.26.11
vscode-xml|red|0.25.0
vscode-yaml|red|1.13.0
LiveServer|rit|5.7.9
vscode-xml-complete|rog|0.3.0
rust-analyzer|rus|0.3.1541
vs-code-prettier-eslint|rve|5.1.0
laravel-artisan|rya|0.0.31
vscode-fileutils|sle|3.10.1
vscode-counter|uct|3.2.1
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-boot-dev-pack|vmw|0.2.1
vscode-concourse|vmw|1.46.0
vscode-manifest-yaml|vmw|1.46.0
vscode-spring-boot|vmw|1.46.0
keyoti-changeallendoflinesequence|vs-|0.0.3
vscode-java-debug|vsc|0.51.0
vscode-java-dependency|vsc|0.23.0
vscode-java-pack|vsc|0.25.11
vscode-java-test|vsc|0.39.0
vscode-maven|vsc|0.41.0
vscode-spring-boot-dashboard|vsc|0.13.1
vscode-spring-initializr|vsc|0.11.2
vscode-icons|vsc|12.4.0
vuetify-vscode|vue|0.2.0
jinja|who|0.0.8
linecount|yyc|0.1.7
markdown-all-in-one|yzh|3.5.1
json|Zai|2.0.2

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30610678
3biah626:30602489
pyind779:30671433
f6dab269:30613381
vscrp:30673768
pythonsymbol12:30671437
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30730052
showsbindicator:30760979
pythongtdpathcf:30739705
dh2dc718:30763024
pythonnosm12tcf:30757130

```

</details>

**Comment by gjsjohnmurray** [comment]
Please use the Start Extension Bisect command to investigate whether the problem is being caused by one of your extensions.

Also, are you using McAfee antivirus?

**Comment by chapus** [comment]
![image](https://github.com/microsoft/vscode/assets/1543053/d223deca-7ae4-4203-b655-42e4986b0670)

I just want to add a screenshot for this ticket. and i will try the Extension Bisect.


```
diff.astextplain.textconv=astextplain
filter.lfs.clean=git-lfs clean -- %f
filter.lfs.smudge=git-lfs smudge -- %f
filter.lfs.process=git-lfs filter-process
filter.lfs.required=true
http.sslbackend=openssl
http.sslcainfo=C:/Program Files/Git/mingw64/etc/ssl/certs/ca-bundle.crt
core.autocrlf=true
core.fscache=true
core.symlinks=false
pull.rebase=false
init.defaultbranch=master
core.editor="C:\Users\PC\AppData\Local\Programs\Microsoft VS Code\bin\code" --wait
credential.helper=manager-core
```

I was reading there is something that is slowing git using windows credentials manager, but idk.

**Comment by lszomoru** [comment]
@chapus, it would be very interesting to see if you can use extension bisect to try to limit the number of extension while you are still able to reproduce the problem. This is needed as the git extension exposes API that other extensions can call into hence appearing unresponsive.

**Comment by chapus** [comment]
@lszomoru i found a warning in vs code git output: "warning: git-credential-manager-core was renamed to git-credential-manager", so i found in my %USERPROFILE%/.gitconfig removed all my helpers for credentials and now my git at least yesterday is not slow.  I need to test more but i see a difference now.

I found this: https://github.com/git-ecosystem/git-credential-manager/issues/352

I removed

```
[credential]
	helper = 
	helper = git-credential-manager-core
	helper = manager-core
```

In my case i had only one helper so I removed the entire section.
